task-1/task1.js

```diff
@@ -0,0 +1,10 @@
+const prompt = "\nType something:\n";
+
+process.stdout.write(prompt)
+
+process.stdin.on("data", (buffer) => {
+  const string = buffer.toString().split("");
+  string.reverse();
+  process.stdout.write("\nReversed: \x1b[32m " + string.join("") + " \x1b[0m \n\n")
```

Comment:
what are these \x1b[32m ? please define some constants and user them instead, it's confusing
e.g. const GREEN_COLOR = '\x1b[32m'

Reply:
done